builtin/providers/aws/resource_aws_route53_zone.go

```diff
@@ -87,9 +90,38 @@ func resourceAwsRoute53ZoneRead(d *schema.ResourceData, meta interface{}) error
 		return err
 	}
 
+	// get tags
+	req := &route53.ListTagsForResourceRequest{
+		ResourceID:   aws.String(d.Id()),
+		ResourceType: aws.String("hostedzone"),
+	}
+
+	resp, err := r53.ListTagsForResource(req)
```

Comment:
Correct, it returns a `route53.ResourceTagSet` which may have a nil set of `tags`.